lib/registries/addon/drafts/draft/page/template.hbs

```diff
@@ -23,147 +23,153 @@
                 <LoadingIndicator @dark={{true}} />
             </div>
         {{else}}
-            <OsfLayout @backgroundClass={{local-class 'ContentBackground'}} as |layout|>
```

Comment:
I feel the changes here, pollute the PR? Just adding the component and some tests is enough. 🤔